doc/cephadm/monitoring.rst

```diff
@@ -1,9 +1,9 @@
 Monitoring Stack with Cephadm
 =============================
 
-The Ceph dashboard makes use of prometheus, grafana, and related tools
-to store and visualize detailed metrics on cluster utilization and
-performance.  Ceph users have three options:
+The Ceph dashboard uses `Prometheus <https://prometheus.io/>`_, `Grafana
```

Comment:
In the dashboard documentation we write the word 'dashboard' with a capital 'D' when mentioning together with 'Ceph' - 'Ceph Dashboard'. We may want to do the same here?

```suggestion
The Ceph Dashboard uses `Prometheus <https://prometheus.io/>`_, `Grafana
```

Reply:
Actually, I would then drop the leading "the", too:
```suggestion
Ceph Dashboard uses `Prometheus <https://prometheus.io/>`_, `Grafana
```